spec/controllers/state_file/questions/ny_sales_use_tax_controller_spec.rb

```diff
@@ -0,0 +1,25 @@
+require "rails_helper"
+
+RSpec.describe StateFile::Questions::NySalesUseTaxController do
+  let(:intake) { create :state_file_ny_intake }
+  before do
+    session[:state_file_intake] = intake.to_global_id
+  end
+
+  describe "#update" do
+    # use the return_to_review_concern shared example if the page
+    # should skip to the review page when the return_to_review param is present
+    # requires form_params to be set with any other required params
+    it_behaves_like :return_to_review_concern do
```

Comment:
in the other similar example we have, an intake factory is passed in. but at least some of these require additional fields to be set on the intake, so i'm creating it and setting it in the session in here instead